template/README.md

```diff
@@ -1,137 +1,157 @@
-
-### Accessing Consul, Vault, Nomad, and MinIO
-The default box will start Nomad, Vault, Consul, and MinIO bound on loopback and advertising on the ip `10.0.3.10`, which should be available on your local machine.
-Portforwarding for nomad on port `4646` should bind to `127.0.0.1` and should allow you to use the nomad binary to post jobs directly. Consul and Vault has also been portforwarded, and are also available on `127.0.0.1` on port `8500` and `8200` respectively.
-Minio is started on port `9000` and sharing /vagrant (your repo) from within the vagrant box.
-- Nomad ui is available on [http://10.0.3.10:4646](http://10.0.3.10:4646) and all links to services should work.
-- Consul ui is available on [http://10.0.3.10:8500](http://10.0.3.10:8500)
-- Vault ui is available on [http://10.0.3.10:8200](http://10.0.3.10:8200)
-- minio ui is available on [http://10.0.3.10:9000](http://10.0.3.10:9000) (minioadmin:minioadmin)
-
-### Pre and post hashistack procedure
+2. `vagrant/.env` - _user provided_ - variables override, see [Pre-packaged Configuration Switches](#pre-packaged-configuration-switches) for details
+3. `vagrant/.env_override` - _system provided_ - variables are overridden for test purposes
+4. `vagrant/dev/vagrant/conf/pre_ansible.sh` - _user provided_ - script running before ansible bootstrap procedure, [details](dev/vagrant/conf/pre_bootstrap/README.md)
+5. `vagrant/dev/vagrant/conf/pre_bootstrap/*.yml` - _user provided_ - pre bootstrap tasks, running before hashistack software starts, [details](dev/vagrant/conf/README.md)
+6. `/etc/ansible/bootstrap.yml` - _preloaded_ - verify ansible variables and software configuration, run hashistack software and MinIO, & verify that it started correctly
+7. `vagrant/conf/post_bootstrap/*.yml` - _user provided_ - poststart scripts, running after hashistack software has started, [details](dev/vagrant/conf/pre_bootstrap/README.md)
+8. `vagrant/dev/conf/pre_ansible.sh` - _user provided_ - script running after ansible bootstrap procedure, [details](dev/vagrant/conf/README.md)
+9. `vagrant/ansible/*.yml` - _user provided_ - ansible tasks included in playbook, see [Pre-packaged Configuration Switches](#pre-packaged-configuration-switches) for details
+
+---
+
+### Pre and Post Hashistack Startup Procedure
 You may change the hashistack configuration or add additional pre and post steps to the startup procedure to match your needs.
 Detailed documentation in [dev/vagrant/conf/README.md](dev/vagrant/conf/README.md)
 
-### Pre packaged configuration switches
+### Pre-packaged Configuration Switches
```

Comment:
Just `NB`
We probably need to follow one name for such functionality. In root documentation, the name has not been explicitly chosen (toggles, switches, ...). If we go for switches, it is better to name them the same everywhere.
Link in root https://github.com/fredrikhgrelland/vagrant-hashistack#configuration